SwiftVocabulary/SwiftVocabulary/VocabularyWord.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  VocabularyWord.swift
+//  SwiftVocabulary
+//
+//  Created by Miguel Marquez on 5/11/20.
+//  Copyright © 2020 Miguel Marquez. All rights reserved.
+//
+
+import Foundation
+
+struct VocabularyWord {
+    let word: String
+    let definition: String
+
+    init(word: String, definition: String) {
```

Comment:
Unlike classes, Structs already come with a built in initializer. So you don't need this.